docset/winserver2022-ps/hyper-v/Set-VMGpuPartitionAdapter.md

```diff
@@ -0,0 +1,427 @@
+---
+description: Configures host partitionable GPU to the number of partitions supported by the manufacturer
+external help file: Microsoft.HyperV.PowerShell.Cmdlets.dll-Help.xml
+Module Name: Hyper-V
+ms.date: 09/22/2022
+online version: https://learn.microsoft.com/en-us/powershell/module/hyper-v/set-vmgpupartitionadapter?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+
+---
+
+# Set-VMGpuPartitionAdapter
+
+## SYNOPSIS
+Configures host partitionable GPU to the number of partitions supported by the manufacturer
+
+## SYNTAX
+
+### VMName (Default)
+```
+Set-VMGpuPartitionAdapter [-CimSession <CimSession[]>] [-ComputerName <String[]>]
+ [-Credential <PSCredential[]>] [-VMName] <String[]> [-Passthru] [-AdapterId <String>]
+ [-MinPartitionVRAM <UInt64>] [-MaxPartitionVRAM <UInt64>] [-OptimalPartitionVRAM <UInt64>]
+ [-MinPartitionEncode <UInt64>] [-MaxPartitionEncode <UInt64>] [-OptimalPartitionEncode <UInt64>]
+ [-MinPartitionDecode <UInt64>] [-MaxPartitionDecode <UInt64>] [-OptimalPartitionDecode <UInt64>]
+ [-MinPartitionCompute <UInt64>] [-MaxPartitionCompute <UInt64>] [-OptimalPartitionCompute <UInt64>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### VMObject
+```
+Set-VMGpuPartitionAdapter [-VM] <VirtualMachine[]> [-Passthru] [-AdapterId <String>]
+ [-MinPartitionVRAM <UInt64>] [-MaxPartitionVRAM <UInt64>] [-OptimalPartitionVRAM <UInt64>]
+ [-MinPartitionEncode <UInt64>] [-MaxPartitionEncode <UInt64>] [-OptimalPartitionEncode <UInt64>]
+ [-MinPartitionDecode <UInt64>] [-MaxPartitionDecode <UInt64>] [-OptimalPartitionDecode <UInt64>]
+ [-MinPartitionCompute <UInt64>] [-MaxPartitionCompute <UInt64>] [-OptimalPartitionCompute <UInt64>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+### Object
+```
+Set-VMGpuPartitionAdapter [-VMGpuPartitionAdapter] <VMGpuPartitionAdapter[]> [-Passthru]
+ [-MinPartitionVRAM <UInt64>] [-MaxPartitionVRAM <UInt64>] [-OptimalPartitionVRAM <UInt64>]
+ [-MinPartitionEncode <UInt64>] [-MaxPartitionEncode <UInt64>] [-OptimalPartitionEncode <UInt64>]
+ [-MinPartitionDecode <UInt64>] [-MaxPartitionDecode <UInt64>] [-OptimalPartitionDecode <UInt64>]
+ [-MinPartitionCompute <UInt64>] [-MaxPartitionCompute <UInt64>] [-OptimalPartitionCompute <UInt64>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Set-VMGpuPartitionAdapter** cmdlet Configures host partitionable GPU to the number of partitions supported by the manufacturer.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Set-VMHostPartitionableGpu -ComputerName �SampleHost� -partitioncount 8
+```
```

Comment:
This example has invalid characters and the `-partitioncount` parameter is not valid.